client/src/components/pages/application/Project/index.js

```diff
@@ -1,5 +1,139 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { TextareaAutosize } from '@material-ui/core';
+import TextField from '../../../common/TextField';
+import Limitation from '../../../common/limitation';
+import Typography from '../../../common/Typography';
+import backGround from '../../../../assets/images/backGround.svg';
+import Button from '../../../common/Button';
+import FProjectValidation from '../../../../utils/application/ProjectValidation';
 
-const Project = () => <h1>Project</h1>;
+import './style.css';
+
+const getUserID = async () => {
+  const { data } = await axios.get('/api/v1/isUser');
+  return data;
+};
+const projectData = async (ID) => {
+  const {
+    data: { project },
+  } = await axios.get(`/api/v1/project/${ID}`);
+  return project;
+};
+const Project = () => {
+  const [projectTitle, setProjectTitle] = useState('');
+  const [projectDesc, setProjectDesc] = useState('');
+  const [githubLink, setGithubLink] = useState('');
+  const [message, setMessage] = useState([]);
+  const [UserId, setId] = useState('');
+  const [userName, setName] = useState('');
+  const history = useHistory();
+
+  const throwMessage = (msg) => setMessage(msg);
+
+  useEffect(() => {
+    getUserID().then((data) => {
+      if (data.message === 'you are authorized') {
+        setId(data.userId);
+      }
+    });
+    if (UserId) {
+      axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
+        setGithubLink(user.githubLink);
+        setName(user.fullName);
+        projectData(user.projectId).then((project) => {
+          setProjectTitle(project.projectName);
+          setProjectDesc(project.projectInstructions);
+        });
+      });
+    }
+  }, [UserId]);
+
+  // useEffect(() => {
+  //   axios.get(`/api/v1/project/${projectId}`).then(({ data: { project } }) => {
+  //     setProjectTitle(project.projectName);
+  //     setProjectDesc(project.projectInstructions);
+  //   });
+  // });
+
+  const Next = () => {
+    FProjectValidation({ githubLink })
+      .then(() =>
+        axios.patch(`/api/v1/applicants/${UserId}`, {
+          githubLink,
+        })
+      )
+      .then(() => history.push('/submit'))
+      .catch(({ errors }) => throwMessage(errors));
+  };
+
+  return (
+    <div className="Container_page">
+      <Helmet>
+        <title>Final Project</title>
+      </Helmet>
+      <img src={backGround} alt="backGround" className="backGround" />
+      <div className="text_Welcome">
+        <Typography variant="h3" color="default">
+          Welcome, {userName}
+        </Typography>
+      </div>
```

Comment:
we didn't need Welcome here, we agreed that we want it only in the steps page . so just delete it from here